website/views/shared/profiles/achievements.jade

```diff
@@ -146,8 +146,8 @@ div(ng-if=':: profile.achievements.originalUser')
   
 div(ng-if='profile.achievements.challenges || user._id == profile._id')
   // This is a very strange icon to use. revisit
-  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges')
-  div(ng-class='{muted: !profile.achievements.challenges}')
+  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges.length')
```

Comment:
It's not required for this PR, but if you're feeling in the mood to do something tedious like that, I won't stop you :wink: